config/config.go

```diff
@@ -39,3 +41,92 @@ func DetermineUseLiveData(logger golog.Logger, liveData *bool, sensors []string)
 	}
 	return useLiveData, nil
 }
+
+// AttrConfig describes how to configure the SLAM service.
+type AttrConfig struct {
+	Sensors             []string          `json:"sensors"`
+	ConfigParams        map[string]string `json:"config_params"`
+	DataDirectory       string            `json:"data_dir"`
+	UseLiveData         *bool             `json:"use_live_data"`
+	DataRateMsec        int               `json:"data_rate_msec"`
+	MapRateSec          *int              `json:"map_rate_sec"`
+	InputFilePattern    string            `json:"input_file_pattern"`
+	Port                string            `json:"port"`
+	DeleteProcessedData *bool             `json:"delete_processed_data"`
+	Dev                 bool              `json:"dev"`
```

Comment:
Please comment on https://viam.atlassian.net/browse/RSDK-1684, that the dev flag needs to be removed from the slam repo.